Microsoft Defender/microsoft-defender-for-endpoints/_meta/smart-descriptions.json

```
@@ -1,6 +1,17 @@
 [
   {
-    "value": "Technique {threat.technique.name} with {microsoft.defender.threat.severity} detected",
+    "value": "{service.name} has raised alert {microsoft.defender.alert.id}.",
```

```suggestion
    "value": "{service.name} has raised alert {microsoft.defender.alert.id}",
```
I think that we don't set dots at the end of the smart descriptions